explanations/71/en.md

```
@@ -0,0 +1,74 @@
+# Problem 71: Simplify Path
+**Difficulty:** Medium  
+**Link:** https://leetcode.com/problems/simplify-path/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)